iam-role/Acornfile

```diff
@@ -0,0 +1,120 @@
+name:        "AWS IAM Role"
+description: "AWS Identity and Access Management (IAM) Role"
+info:        localData.info
+
+args: {
+	// Name of the Role to create. The default is auto-generated.
+	roleName: ""
+	// The ARN of the principal that will be allowed to assume the role. Required.
+	trustedArn: ""
+	// Extra tags to place on the created Role.
+	tags: {}
+	// The IAM policy that will be attached to the Role as an inline policy. Must be in JSON format. Required.
+	policy: {}
+	// The maximum session duration (in minutes) allowed. Default and minimum is 60.
+	maxSessionDurationMinutes: 60
+	// The path under which the Role will be created. Default is "/".
+	path: "/"
+	// External IDs to use when assuming the role, as a comma-separated list (i.e. "one,two,three"). Default is none.
+	externalIds: ""
+}
+
+services: role: {
+	name: "AWS IAM Role"
+	generated: job: "apply"
+}
+
+jobs: apply: {
+	build: {
+		context: "."
+		additionalContexts: {
+			common: "../libs"
+			utils:  "../utils"
+		}
+	}
+	files: {
+		"/app/config.json": std.toJSON(args)
+		"/app/policy.json": std.toJSON(args.policy)
```

Comment:
This seems unnecessary since it will be dumped as a json object to the config.json file. You should be able to just unmarshal it into the stackProps object. In the go code it seems like it is just a map[string]interface{} which should be doable.